client/my-sites/posts/post.jsx

```diff
@@ -1,37 +1,37 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	ReactCSSTransitionGroup = require( 'react-addons-css-transition-group' ),
-	classNames = require( 'classnames' );
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var Card = require( 'components/card' ),
-	Gridicon = require( 'components/gridicon' ),
-	PostRelativeTimeStatus = require( 'my-sites/post-relative-time-status' ),
-	PostControls = require( './post-controls' ),
-	PostHeader = require( './post-header' ),
-	PostImage = require( '../post/post-image' ),
-	PostExcerpt = require( 'components/post-excerpt' ),
-	PostTotalViews = require( 'my-sites/posts/post-total-views' ),
-	utils = require( 'lib/posts/utils' ),
-	updatePostStatus = require( 'lib/mixins/update-post-status' ),
-	analytics = require( 'lib/analytics' );
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import PostRelativeTimeStatus from 'my-sites/post-relative-time-status';
+import PostControls from './post-controls';
+import PostHeader from './post-header';
+import PostImage from '../post/post-image';
+import PostExcerpt from 'components/post-excerpt';
+import PostTotalViews from 'my-sites/posts/post-total-views';
+import utils from 'lib/posts/utils';
+import updatePostStatus from 'lib/mixins/update-post-status';
+import analytics from 'lib/analytics';
 
 import Comments from 'reader/comments';
 
 function recordEvent( eventAction ) {
 	analytics.ga.recordEvent( 'Posts', eventAction );
 }
 
-function checkPropsChange( nextProps, propArr ) {
-	var i, prop;
+function checkPropsChange( currentProps, nextProps, propArr ) {
+	let i, prop;
```

Comment:
instead of change var to let can we just inline the let. So it reads. 
`for( let i =0; `
and 
`const prop = propArr[ i]` 
instead.